contrib/splitup.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -xe
+
+# VARIABLES
+BASEDIR=$(dirname "$0")
+LOCAL_BRANCH=$(cd $BASEDIR && git rev-parse --abbrev-ref HEAD)
+
+
+# FUNCTIONS
+function check_existing_remote {
+  if ! git remote show $1 &> /dev/null; then
```

Comment:
This conditional is broken-- it always evaluates to true so the `git add` line causes a fatal error if the remote already exists, stopping the script since it is in `set -e`

Maybe use:
```
function check_existing_remote {
  git remote get-url $1 2>&1 > /dev/null
  if [[ $? -ne 0 ]]; then
    git remote add $1 git@github.com:/ceph/ansible-ceph-$1.git
  fi
}
```

Note this breakage shouldn't affect the execution of this script against a fresh repo in a CI build, but someone else trying to use this against the same repo multiple times would run into problems.